segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/TableSegmentStatsRecorderImpl.java

```diff
@@ -67,57 +69,42 @@ public void close() {
 
     @Override
     public void createTableSegment(String tableSegmentName, Duration elapsed) {
-        getCreateSegment().reportSuccessEvent(elapsed);
+        this.createSegment.reportSuccessEvent(elapsed);
     }
 
     @Override
     public void deleteTableSegment(String tableSegmentName, Duration elapsed) {
-        getDeleteSegment().reportSuccessEvent(elapsed);
-        String[] segmentTags = segmentTags(tableSegmentName);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_UPDATE_CONDITIONAL, segmentTags);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_UPDATE, segmentTags);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_REMOVE_CONDITIONAL, segmentTags);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_REMOVE, segmentTags);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_GET, segmentTags);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_ITERATE_KEYS, segmentTags);
-        getDynamicLogger().freezeCounter(MetricsNames.TABLE_SEGMENT_ITERATE_ENTRIES, segmentTags);
+        this.deleteSegment.reportSuccessEvent(elapsed);
```

Comment:
I completely removed these metrics. We were reporting both global metrics and per-segment metrics. However this seems to be overkill, especially for Table Segments. To avoid having to do unnecessary work, I decided to only keep the global metrics and get rid of the per-segment ones. Only the per-segment ones required DynamicLogger, and the other ones are properly closed when this class is closed.

FYI, "Freezing" is only needed for Dynamic Loggers. The "non-dynamic" ones should be closed when they are no longer needed.